src/Components/Calculator.js

```diff
@@ -1,34 +1,43 @@
 import React, { Component } from 'react';
+import calculate from '../logic/calculate';
 
 class Calculator extends Component {
   constructor(props) {
     super(props);
-    this.state = {};
+    this.state = {
+      total: null,
+      next: null,
+      operation: null,
+    };
   }
 
+  handleClick = (e) => this.setState((obj) => calculate(obj, e.target.innerText));
+
   render() {
+    const { total, next, operation } = this.state;
+    const result = `${total}${operation}${next}`.replace(/null/g, '').replace(/undefined/g, '');
     return (
       <div className="calculator-container">
-        <div className="output">0</div>
-        <button className="ac" type="button">AC</button>
-        <button className="plus-minus" type="button">+/-</button>
-        <button className="percentage" type="button">%</button>
-        <button className="division" type="button">/</button>
-        <button className="seven" type="button">7</button>
-        <button className="eight" type="button">8</button>
-        <button className="nine" type="button">9</button>
-        <button className="product" type="button">*</button>
-        <button className="four" type="button">4</button>
-        <button className="five" type="button">5</button>
-        <button className="six" type="button">6</button>
-        <button className="minus" type="button">-</button>
-        <button className="one" type="button">1</button>
-        <button className="two" type="button">2</button>
-        <button className="three" type="button">3</button>
-        <button className="plus" type="button">+</button>
-        <button className="zero" type="button">0</button>
-        <button className="dot" type="button">.</button>
-        <button className="equal" type="button">=</button>
+        <div className="output">{result === '' ? '0' : result}</div>
+        <button className="ac" type="button" onClick={this.handleClick}>AC</button>
+        <button className="plus-minus" type="button" onClick={this.handleClick}>+/-</button>
+        <button className="percentage" type="button" onClick={this.handleClick}>%</button>
+        <button className="division" type="button" onClick={this.handleClick}>/</button>
+        <button className="seven" type="button" onClick={this.handleClick}>7</button>
+        <button className="eight" type="button" onClick={this.handleClick}>8</button>
+        <button className="nine" type="button" onClick={this.handleClick}>9</button>
+        <button className="product" type="button" onClick={this.handleClick}>*</button>
```

Comment:
- [ ] I also noticed another bug 🐛 when I attempted to use the `multiply *` operator in your app. Please take a look at the console error below for more information 👇 
![image](https://user-images.githubusercontent.com/46839548/206443852-bcd647b1-5152-4281-9839-f3df7c1df505.png)